modules/tinymce/src/core/test/ts/browser/NotificationManagerTest.ts

```diff
@@ -57,6 +60,8 @@ describe('browser.tinymce.core.NotificationManagerTest', () => {
 
       it('TBA: Should not add duplicate text message', () => {
         const editor = hook.editor();
+        resetNotifications();
```

Comment:
Could this be a beforeEach or would that be harder to understand.

Reply:
Not every test needs this to be reset (the first test for example), so using a `beforeEach` would break things. I did wonder if this maybe should be 2 separate tests though, but ended up deciding on this (though happy to do something else if others prefer).